src/components/categories/CategoriesPic.tsx

```diff
@@ -2,18 +2,72 @@ import * as React from 'react';
 import { Container } from '@mui/system';
 import CategoryImage from '../categoryImages/CategoryImage';
 import {CategoryList, TypographyHeading} from './CategoriesPicStyle'
+import axios from 'axios'
+import {baseUrl , token } from "../../API/index"
+import {useEffect, useState} from "react";
+import Frame from '../../assets/frame2.png'
 
+
+const initialState = [
+    {
+        id : 1,
+        img : Frame,
+        title: ''
+    },
+    {
+        id : 2,
+        img : Frame,
+        title: ''
+    },
+    {
+        id : 3,
+        img : Frame,
+        title: ''
+    },
+    {
+        id : 4,
+        img : Frame,
+        title: ''
+    },
+    {
+        id : 5,
+        img : Frame,
+        title: ''
+    },
+    {
+        id : 6,
+        img : Frame,
+        title: ''
+
+    },
+]
 const Categories= () => {
+    const [data , setData] = useState(initialState)
```

Comment:
You don't need the initial state to be static, if you need to show something to the user while the request is loading, you can show some skeltons 
https://mui.com/material-ui/react-skeleton/